### PR TITLE
Proposed changes

### PR DESCRIPTION
**Pull request by arturl**
Use first person plural, plus other wording changes.

Note a couple of comments that need follow-up before we publish.

### CODE REVIEW

File: content/blog/copilot-lessons/index.md [hunk 1]
```diff
@@ -63,22 +61,24 @@ As Copilot grew, we broke it into smaller pieces we call skills. Each skill does
...
 
 Refining this routing system revealed another opportunity: streamlining the Debug button workflow.
 
 ## Debug Dispatch
 
 ![Before and After](optimize.png)
 
+<!-- We have an issue here with the picture: LLM does not call the API. LLM used to make up the URL (occasionally hallucinating), then Copilot would call that API and fed the results back the LLM for summarization. This is a very important point about how tool invocation work, we must get it right -->
```
Comment (arturl) [inline content/blog/copilot-lessons/index.md:72]: Issue with the picture, cc @adamgordonbell

File: content/blog/copilot-lessons/index.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ But while minimizing the LLM workload helped with efficiency, we soon faced an e
 
 Large language models excel at generating well-structured, grammatically correct output. They makes neat tables, tell good stories, and generally sound confident. That's what makes them dangerous because this polished presentation can mask underlying flaws in the information itself, creating a false sense of confidence for users.
 
-One of our early testers, Pablo, encountered this firsthand. He posed a query to Pulumi Copilot, asking for a summary of resources within a specific project. The response he received was impeccably formatted, neatly categorizing resources by type and providing counts for each. It *looked* right, and for us humans sometimes looking right carries a lot of weight.
+One of our early testers, Pablo, <!-- TODO: ask Pablo --> encountered this firsthand. He posed a query to Pulumi Copilot, asking for a summary of resources within a specific project. The response he received was impeccably formatted, neatly categorizing resources by type and providing counts for each. It *looked* right, and for us humans sometimes looking right carries a lot of weight.
```
Comment (arturl) [inline content/blog/copilot-lessons/index.md:93]: Must ask Pablo if he is OK to be mentioned, or leave the name out. cc: @adamgordonbell

File: content/blog/copilot-lessons/index.md (hunk 1 above)
Comment (adamgordonbell) [reply content/blog/copilot-lessons/index.md:72]: yeah, I'm changing it to Copilot. so Copilot calls the api, calls the LLM ect.

File: content/blog/copilot-lessons/index.md (hunk 2 above)
Comment (adamgordonbell) [reply content/blog/copilot-lessons/index.md:93]: @sicarul Thoughts on being in here?

**Review by adamgordonbell — APPROVED** [review]

File: content/blog/copilot-lessons/index.md (hunk 2 above)
Comment (sicarul) [reply content/blog/copilot-lessons/index.md:93]: Hey team! I'm fine with this, thanks for checking!

**Pull request merged**